docs/developer/tutorials/logs/submit-logs-from-http.md

```diff
@@ -0,0 +1,142 @@
+# Submit Logs from HTTP API
+
+## Getting Started
+
+This tutorial assumes you have done the following:
+
+- [set up your environment](../../index.md#getting-started)
+- read the [logs crawler documentation](../../base/logs-crawlers.md)
+- read about the [HTTP capabilities](../../base/http.md) of the base class
+
+Let's say we are building an integration for an API provided by *ACME Inc.*
+Run the following command to create the scaffolding for our integration:
+
+```
+ddev create ACME
+```
+
+This adds a folder called `acme` in our `integrations-core` folder.
+The rest of the tutorial we will spend in the `acme` folder.
+```
+cd acme
+```
+
+In order to spin up the integration in our scaffolding, if we add the following to `tests/conftest.py`:
+
+```python
+@pytest.fixture(scope='session')
+def dd_environment():
+    yield {'tags': ['tutorial:acme']}
+```
+
+Then run:
+```
+ddev env start acme py3.11 --base
+```
+
+## Define an Agent Check
+
+We start by registering an implementation for our integration.
+At first it is empty, we will expand it step by step.
+
+Open `datadog_checks/acme/check.py` in our editor and put the following there:
+
+```python
+from datadog_checks.base.checks.logs.crawler.base import LogCrawlerCheck
+
+
+class AcmeCheck(LogCrawlerCheck):
+    __NAMESPACE__ = 'acme'
+```
+
+Now we'll run something we will refer to as *the check command*:
+```
+ddev env agent acme py3.11 check
+```
+
+We'll see the following error:
+```
+Can't instantiate abstract class AcmeCheck with abstract method get_log_streams
+```
+
+We need to define the `get_log_streams` method.
+As [its docs say](../../base/logs-crawlers.md#datadog_checks.base.checks.logs.crawler.base.LogCrawlerCheck.get_log_streams) it must return an iterator over `LogStream` subclasses.
+That's what the next section is about.
+
+## Define a Stream of Logs
+
+In the same file let's add a `LogStream` subclass and return it (wrapped in a list) from `AcmeCheck.get_log_streams`:
+
+```python
+from datadog_checks.base.checks.logs.crawler.base import LogCrawlerCheck
+from datadog_checks.base.checks.logs.crawler.stream import LogStream
+
+class AcmeCheck(LogCrawlerCheck):
+    __NAMESPACE__ = 'acme'
+
+    def get_log_streams(self):
+        return [AcmeLogStream(check=self, name='ACME log stream')]
+
+class AcmeLogStream(LogStream):
+    """Stream of Logs from ACME"""
+```
+
+Every time we modify the source code, make sure to restart the environment:
+
+```
+ddev env stop acme py3.11 && ddev env start acme py3.11 --base
```

Comment:
```suggestion
ddev env reload acme py3.11
```

Reply:
If we're using `--base` I think we can omit this whole part since it hot reloads IIRC?

Reply:
Just tested and yea looks like `--dev` makes this unnecessary!